core/lib/services/checks.ts

```diff
@@ -34,6 +34,15 @@ export function isString(value: any): value is string {
 	return typeof value === 'string';
 }
 
+/**
+ * an array type guard
+ * @param value the value to check
+ * @returns true if the value is an array
+ */
+export function isArray(value: any) {
```

Comment:
```suggestion
export function isArray(value: any): value is any[] {
```
(cf https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates)